pkgs/applications/misc/zola/default.nix

```diff
@@ -32,9 +32,9 @@ rustPlatform.buildRustPackage rec {
   buildInputs = [
     openssl
     oniguruma
-  ] ++ lib.optional stdenv.isDarwin [
-    CoreServices
     libsass
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
We can use `lib.optional` here since it's a single item, but I'm not sure if that's idiomatically correct.

Reply:
In my opinion using `lib.optionals` unconditionally is easier, than having a mixture consisting of `lib.optional` and `lib.optionals` in different incarnations (e.g. `pkgs/development/libraries/gstreamer/base/default.nix`).

Reply:
``lib.optional CoreServices`` or ``lib.optionals [ CoreServices ]``